src/models/tiles/text/text-content.ts

```diff
@@ -78,7 +78,8 @@ export const TextContentModel = TileContentModel
     exportJson(options?: ITileExportOptions) {
       const value = self.asSlate();
       const html = value ? slateToHtml(value) : "";
-      const exportHtml = html.split("\n").map((line, i, arr) => `    "${line}"${i < arr.length - 1 ? "," : ""}`);
+      const exportHtml = html.split("\n")
+        .map((line, i, arr) => `    "${line.replaceAll(`"`, `\\"`)}"${i < arr.length - 1 ? "," : ""}`);
```

Comment:
I remain concerned that this simplistic approach to escaping will turn out to be insufficient. Perhaps in the short term a comment to at least indicate to our future selves that there's a potential issue here would be worthwhile.

Reply:
I did look into libraries to handle this for a bit, and they all seemed like overkill for what was needed.

That said, while experimenting more with this to make sure the simple `replaceAll` I was using was sufficient, I was surprised and confused to discover that `"` was now being converted to `&quot;` before the `replaceAll` was even running. I tracked it down to a call to `renderToStaticMarkup` in `slateToHtml` in `slate-editor`. I still have no idea why this wasn't happening previously, but is happening now. In any case, `&quot;` seems to work ok in the curriculum json, so I was able to remove this extra layer of processing.

However, experimenting more did show that backslashes make the curriculum json unhappy, so I added an extra step to escape them. I created a named function in a new `string-utils` file, so I hope this is clear enough to not require an explicit comment explaining the situation, but let me know if you think I should add one.